ee/tabby-db/migrations/07-email-service-credentials/down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE smtp_info;
```

Comment:
```suggestion
DROP TABLE email_service_credential;
```

Not sure why this is not captured by migration test